PHP/tests/syntax_test_php.php

```diff
@@ -1935,12 +1935,12 @@ function generate2()
 //       ^^^ entity.name.tag.heredoc
 one: two
 //^^^^^^ meta.embedded.yaml source.yaml
-//^ string.unquoted.plain.out entity.name.tag
+//^ meta.mapping.key string
 // ^       punctuation.separator.key-value.mapping
-//   ^^^ string.unquoted.plain.out
+//   ^^^ string
 three: "$four"
 //^^^^^^^^^^^^ meta.embedded.yaml source.yaml
-//^^^ string.unquoted.plain.out entity.name.tag
+//^^^ meta.mapping.key string
 //   ^       punctuation.separator.key-value.mapping
```

Comment:
Shouldn't it rather be `punctuation.separator.mapping.key-value`? Also compare the related `punctuation.separator.mapping.pair`.

Reply:
Note that it *was* `punctuation.separator.mapping.key-value` before, and it got changed to `punctuation.separator.key-value.mapping` in https://github.com/sublimehq/Packages/commit/60e7c391deb32d1d39b5add199f827c7d3bb8050 for a reason. Could it be that PackageDev was just not updated with the newer scope? It would be great to have a consistent scope for key-value separators between syntaxes, and the only ones which still seem to be left are JSON and Python. Putting the more specific `mapping` at the third level would require color schemes to target at least two scopes for key-value separators, unless you want to introduce this `mapping` at the third level also for syntaxes like CSS and HTML, which don't use the `meta.mapping` scope.

Reply:
The use of one order or the other is split across syntaxes, might as well flip a coin at this point.

Reply:
We should keep it as is with regards to #3228.